var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -27,28 +27,31 @@
 
 class Paraview(Package):
```

Comment:
Nope, that's not the case. I would be interested to here what problems you are seeing.